src/backend/schemas/deployment.py

```diff
@@ -22,49 +21,40 @@ class DeploymentWithModels(BaseModel):
     id: Optional[str] = None
     name: str
     description: Optional[str] = None
+    deployment_class_name: Optional[str] = Field(exclude=True, default="")
+    env_vars: Optional[List[str]]
     is_available: bool = False
     is_community: Optional[bool] = False
-    env_vars: Optional[List[str]]
-    deployment_class_name: Optional[str] = Field(exclude=True, default="")
     models: list[ModelSimple]
 
     class Config:
         from_attributes = True
 
 
-class Deployment(BaseModel):
-    id: Optional[str] = None
+class DeploymentDefinition(BaseModel):
+    id: str
     name: str
-    models: list[str]
-    is_available: bool = False
-    deployment_class: Optional[Type[Any]] = Field(exclude=True, default=None)
-    env_vars: Optional[List[str]]
     description: Optional[str] = None
-    deployment_class_name: Optional[str] = Field(exclude=True, default="")
-    is_community: Optional[bool] = False
-    default_deployment_config: Optional[Dict[str, str]] = Field(
-        default_factory=dict, exclude=True
-    )
-    kwargs: Optional[dict] = Field(exclude=True, default={})
+    config: Dict[str, str] = {}
```

Comment:
I think we shouldn't display the full configuration with key values in a public area.